pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/PinotHelixResourceManager.java

```diff
@@ -1686,6 +1687,25 @@ private void sendSegmentRefreshMessage(OfflineSegmentZKMetadata segmentZKMetadat
     }
   }
 
+  /**
+   * Gets assigned instances for uploading new segment. This method can be used to detect
```

Comment:
Please update the comment. The intent is not to just to discover misconfigurations. You can remove the second sentence.

Reply:
Done.